Assets/_BForBoss/_Core/Scripts/Debug/DebugWindow.cs

```diff
@@ -120,6 +122,12 @@ private void CreateDebugActionList()
                 {
                     GUILayout.Space(0.15f * _windowRect.height);
                     _playerLifeCycle.IsInvincible = DrawDebugToggle(_playerLifeCycle.IsInvincible, new GUIContent("Player Invincibility"));
+                    _energySystemBehaviour.UseDebugEnergySystemConfig = DrawDebugToggle(_energySystemBehaviour.UseDebugEnergySystemConfig, new GUIContent("Freeze Energy"));
+
+                    if (GUILayout.Button("Give Max Energy"))
+                    {
+                        _energySystemBehaviour.SetMaxEnergy();
```

Comment:
Is there a point to this if you're going to freeze energy anyway?

Reply:
Yup. In the case of testing which values feel the best for costs (i.e. shooting and slow-mo) its nice to be able to test how they feel over and over again without needing to manually accrue the energy.